app/uk/gov/hmrc/individualsincomeapi/connector/IntegrationFrameworkConnector.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
```

Comment:
"IntegrationFramework" in class name here, but "IF" elsewhere? (I prefer shorter 🙂)